### DISCUSSION TITLE
How to import several MenuItems from module migration, but keep the original Menu and MenuItems unchanged,please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#15078] [Q&A]
Hi,
    How to import several MenuItems from module migration , but keep the original Menu and MenuItems unchanged,please?

**Comment by MikeAlhayek** [comment]
@effapp please only use issue if you have a real issue. For questions, please open a discussion instead.


You should be able to do that with `MergeSettings` instead `WithSettings`

**Comment by effapp** [comment]
Hi @MikeAlhayek ,

I use IRecipeMigrator to migration:

public async Task<int> CreateAsync()
{
    await _recipeMigrator.ExecuteAsync("MyMenuItems.Recipe.json", this);
    return 1;
}

There are no MergeSettings and WithSettings fro using this method.

Thanks.

**Comment by effapp** [comment]
Hi：
     Does anyone know?
It would be best to give me a code example.

Thanks.
